blog/2024/03-05-new-ci-new-repo/index.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: New CI, New Repo, A Renewed Push for 10.9.0
+description: There's some things about our repo we need to share as we prepare for 10.9.0
+authors: joshuaboniface
+date: 2024-03-05
+slug: new-ci-new-repo
+tags: [release, infrastructure]
+---
+
+Over the last several weeks, I've been driving a major push to revamp and improve our CI, in an effort to improve both our release workflow, our velocity of releases, and the burden they have on me as release manager. This quick post will detail some of the major changes, how they might affect you as a user, and also, how we're planning to proceed with our 10.9.0 release cycle. Read on for more information!
+
+## The TL;DR
+
+1. We have [a new repository browser UI](https://repo.jellyfin.org) on a new master repository machine, built by new CI, that will hopefully make it nicer to look around and get right to what you need. This is still a bit of a work in progress, so please report any bugs you find to us!
+
+2. For 10.9.0, we will not be producing explicit "beta" releases. Instead, we will test using our new "weekly" unstable builds. Once the master branch is sufficiently stable and good, we will release 10.9.0 directly from master.
+
+3. For 10.9.0, we are dropping non-LTS Ubuntu packages. If you are using a non-LTS version, consider switching to 24.04 (which is LTS) or a Docker install.
+
+4. The 10.9.0 feature freeze (bugfix PRs only after this) will tentatively begin on **Monday, March 18th**. The hope is that all of the above will be ready by then so that obtaining unstable builds for testing will be east.
```

Comment:
```suggestion
4. The 10.9.0 feature freeze (bugfix PRs only after this) will tentatively begin on **Monday, March 18th**. The hope is that all of the above will be ready by then so that obtaining unstable builds for testing will be easy.
```